ts/components/screens/DarkLayout.tsx

```diff
@@ -93,7 +94,7 @@ export default class DarkLayout extends React.Component<Props> {
       <React.Fragment>
         {wrapper(
           <React.Fragment>
-            <VSpacer size={16} />
+            <VSpacer size={24} />
```

Comment:
```suggestion
```
We should delete it and insert it in the specific sections, because this spacer creates an unexpected gap in some dark layouts (see the Fiscal Code screen to test yourself).

(Already working on a PR that improves the Wallet and Profile screens from this point of view)